### ISSUE TITLE
1-01-01 Path structure of Aerosol (particle phase) composition variables

### ISSUE THREAD

**Issue by fstuerzl** [wmo-im/wmds#257]
**Branch**
[add when created]

**Summary and Purpose**
The subdomain \Atmosphere\Aerosol\Composition is further subdivided into eight categories. In the disscussion about issue #168 the this structure was found to be impractical:

> The variables describing the particle phase composition should be organised in an unambiguous way, which is notoriously difficult for chemical species. One of the few examples would be the IUPAC system. However, as we see with the example of "total carbon", observed properties are sometimes impossible to place in the IUPAC structure. One solution could be to have no further structure below the "composition" level. The part of the hierarchy below "Atmosphere\Particle phase\composition" seems to be overstructured.

(Quote: @markusfiebig)

Structure: 

|Domain| Subdomain_1 | Subdomain_2 | Subdomain_3 |
|-|-|-|-|
|Atmosphere | Aerosol | Composition | | 
|Atmosphere | Aerosol | Composition | Inorganic anions | 
|Atmosphere | Aerosol | Composition | Inorganic carbonaceous | 
|Atmosphere | Aerosol | Composition | Inorganic cations | 
|Atmosphere | Aerosol | Composition | Inorganic nitrogen species | 
|Atmosphere | Aerosol | Composition | Major inorganic components | 
|Atmosphere | Aerosol | Composition | Organic anions | 
|Atmosphere | Aerosol | Composition | Organic carbonaceous | 
|Atmosphere | Aerosol | Composition | Trace elements| 

**Stakeholder(s)**
WG ACV

**Proposal**
One possible solution: Remove all subdomains below compostion level.

**Comment by charlesabrock** [comment]
I agree with Markus that we need to minimize the number of subdomains because we quickly get to excessive speciation. With several subdomains, it would take a very specific search to find a relevant measurement. Better to provide a list of all compositions, even if it is long, than require a user to be extremely specific to find relevant data.

**Comment by markusfiebig** [comment]
There seems to be a consensus to reduce the number of subdomains. We need to discuss how many subdomains we can accomodate in a systematic manner. Already subdomain_1 seems to be somewhat heterogeneous in granularity:
Particle phase
gas
Pressure
....

**Comment by fstuerzl** [comment]
Considering the recent discussions about a using tags instead paths:
For particle composition - according to the existing path structure - the following terms need to be as tags.
- Inorganic anions
- Inorganic cations
- Inorganic carbonaceous
- Inorganic nitrogen species
- Major inorganic components
- Organic anions
- Organic carbonaceous
- Trace elements

Please review the list with regard to the discussion in #261.

**Comment by sebvi** [comment]
This list illustrates that we need "organic" and "inorganic" on their own so that we can combine them with other attributes.

**Comment by joergklausen** [comment]
See https://github.com/wmo-im/wmds/wiki/Tags-and-matrices-for-observed-variables for discussion.

**Comment by joergklausen** [comment]
drop (or do not add) major inorganic compoents as a tag.
